pz_be_services/db/crud/crud_chat.py

```
@@ -5,12 +5,12 @@
 
 from .base import CRUDBase
 from ..models import Chat, User, Message
-from schemas.chat import ChatCreate, ChatUpdate
+from schemas.chat import ChatCreate, ChatUpdate, ChatCreateModel
```

Ruff (0.12.2)

`schemas.chat.ChatCreate` imported but unused

Remove unused import: `schemas.chat.ChatCreate`

(F401)